communications.md

```diff
@@ -2,95 +2,62 @@
 
 ## General guidelines
 
-Communication is critical in any organization but in doing so, we want to ensure that we are considerate of other people's production processes and time. Therefore, the focus should be on real problem solving and creative endeavors instead of drowning oneself in processes and tools when communicating with one another.
+Communication is critical to any organization and in doing so, we should be considerate of other people's production processes and time.
+We use a few tools to aid customer and team (inter and intra) communication: Slack, G Suite, Asana and Emails.
```

Comment:
`inter` and `intra` are the same thing. i think you mean `internal` and `external`

Reply:
**Inter**- is a prefix that means between two groups, and **intra**- is a prefix which means within or inside one group. 
Not internal and external

Reply:
Fair enough. I was thinking intranet as internal network :trollface: